.github/workflows/ui.yml

```diff
@@ -53,8 +53,12 @@ jobs:
           node-version: 16
           cache: pnpm
           cache-dependency-path: ui/pnpm-lock.yaml
-      - run: pip install -r ../requirements.txt
-      - run: pip install gradio
+      - name: Install Python
+        uses: actions/setup-python@v3
+        with:
+          python-version: '3.x'
+      - run: python -m pip install --upgrade pip
```

Comment:
Is this still necessary if we are using the updated run (which comes with pip v22)?

Reply:
Think so otherwise you will get (no setup.py detected but found a toml instead) error

Reply:
Let me just test to see!

Reply:
💥 You were right